wire-ios-request-strategy/Resources/be.lproj/Push.strings

```diff
@@ -1,256 +0,0 @@
-// 
```

Comment:
Probably we need a script for the crowdin action to not generate unsupported languages

Reply:
those are not used at all, so do we really need anything here?